.well-known/security.txt

```diff
@@ -0,0 +1,16 @@
+# If you would like to report or fix a bug
+# Create an issue or a pull request on the
+# Northstar Github page
+
+# Northstar Atlas (Master Server)
+https://github.com/R2Northstar/Atlas
+
+# Northstar Launcher
+https://github.com/R2Northstar/NorthstarLauncher
+
+# Northstar Game Scripts
+https://github.com/R2Northstar/NorthstarMods
+
+# Or notrify one of our developers on Discord
+Gecko#7945
+EmmaM#6474
```

Comment:
With d874cc07f58011a52f6ecd0f692e935242b1f504 this file should now be moved to `dist/.well-known/security.txt`

Reply:
Updated